test/fastboot-test.js

```diff
@@ -101,16 +101,15 @@ describe("FastBoot", function() {
       });
   });
 
-  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
+  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function(done) {
```

Comment:
I think since you are returning `resolved` promise it is behaving correctly, it either needs rejections inside `then` or it needs to use `done`
> I think this should be fixed in mocha upstream as well.